### DISCUSSION TITLE
"Request failed with status code 403"

### DISCUSSION THREAD

**Discussion by vinot** [cvat-ai/cvat#4285] [Q&A] [answered]
Hi,

thank you for cvat.org , I try it several times with a browser (firefox and chrome, on linux and mac, many times) and it runs very well and easy (I signed once, at https://cvat.org/auth/login)

But since few days ago I got 'Error: Request failed with status code 403. {"detail":"You do not have permission to perform this action."}.'

First I tried [https://github.com/openvinotoolkit/cvat/issues/198#issuecomment-439316577] and [https://github.com/openvinotoolkit/cvat/issues/639#issue-480009735] (I dont installed Django or anything else, I use it "online" with a browser) with no succesful.

Then I restarted browsers and finally I made a hard reset on mac, but I still get the error:

![Captura de pantalla 2022-02-03 a las 18 51 49](https://user-images.githubusercontent.com/607087/152400155-2abc1fbc-082a-49b9-a710-c737b0551de4.png)

What I'm missing?

**Comment by nmanovic** [comment, marked as answer]
@vinot, cvat.org is a public instance with the limited amount of resources and huge number of users. Previously every registered user could run automatic annotation for a task. Now it is possible only for users with `business` and `admin` roles. You don't have such permissions on cvat.org. Thus you get the 403 error now.

Good news. You can run automatic annotation inside an annotation job for one frame. It will work for `user` role (as most of users have the role on cvat.org).

P.S. We are working on a solution to attach own computing resources. Thus you will be able to overcome the problem and use own computing resources to run automatic annotation on a job, task or project.

**Comment by vinot** [comment]
Understood.

Pls let me ask again, does it means that a local installation [https://openvinotoolkit.github.io/cvat/docs/administration/basics/installation/](url) still uses your hardware? (I tried a local install on my linux with the same error)

thank you

**Comment by nmanovic** [comment]
@vinot , local installation should not suffer from the issue. Please check that you have enough permissions on your local system. For example, admin has no restrictions at all. On your screenshot the error is on cvat.org

**Comment by vinot** [comment]
thank you very much, I'm going to check it

update: install on ubuntu20.04 with cuda works perfect, thank you again

El dv, 4 febr 2022 a les 15:23 Nikita Manovich ***@***.***>
va escriure:
